### ISSUE TITLE
Gutenberg doesn't work with WP Multilang

### ISSUE THREAD

**Issue by Slaver** [VaLeXaR/wp-multilang#69]
Posts are not saved after installing Gutenberg 3.3.0 on site with WP Multilang 2.2.5. I've checked it on empty site after installation WordPress 4.9.7

**Comment by VaLeXaR** [comment]
Hi. Which post type You are trying to translate? Built-in or custom?

**Comment by Slaver** [comment]
> Hi. Which post type You are trying to translate? Built-in or custom?

Default post types

**Comment by VaLeXaR** [comment]
I tested my plugin with last version Gutenberg and all are working fine.

**Comment by Slaver** [comment]
I've made the screencast for displaying the problem https://drive.google.com/file/d/1MFttDYW-rdeXhBwQwtTTEtNv4aGfq91Z/view
Everything works without Gutenberg

**Comment by VaLeXaR** [comment]
Thanks for screencast. Did You try reload page in admin? Does the text changes exist? Or the problem only on front side?

**Comment by Slaver** [comment]
> Did You try reload page in admin? Does the text changes exist? Or the problem only on front side?

Yes, I've tryed to reload — text changes disappeared as well as on front side.

**Comment by VaLeXaR** [comment]
I tested it on the test site and all work fine http://fragile-yak.w6.poopy.life/2017/03/09/hello-world/
You can try yourselve.
http://bit.ly/2n4m5Q1
http://bit.ly/2OxvUCm
http://bit.ly/2OCjyZN

**Comment by VaLeXaR** [comment]
Login: http://fragile-yak.w6.poopy.life/wp-admin/
Username: demo
Password: Z9Q1x0sDc0Hc

**Comment by Anticosti** [comment]
You are right VaLeXaR. 
Just tested and it works perfectly. :)

**Comment by Slaver** [comment]
Thanks! It's very interesting. Maybe I have some problems with my test server. I'll try to debug

**Comment by Slaver** [comment]
I've tested on another server and everything is OK. But I can't still to find problem on my local machine :(

**Comment by Slaver** [comment]
The problem happend only when option 'Use prefix for language by default' is activated.

**Comment by VaLeXaR** [comment]
Only on Your server? Or You tested this bug on a test server?

**Comment by Slaver** [comment]
It seems that only on my server. I've tested on two virtual machines with different versions of Ubuntu and Nginx/PHP-FPM. Maybe I've some problems with routes. For example, when I try to preview post I get Too Many Redirects (WP_DEBUG=true).

**Comment by Anticosti** [comment]
I had this same issue. Thank you @Slaver for the hint.
